buildconfig/Jenkins/Conda/vcvarsall.sh

```diff
@@ -0,0 +1,431 @@
+#!/usr/bin/env bash
```

Comment:
script taken from https://github.com/nathan818fr/vcvars-bash

Does not need thorough review